neutrino.go

```diff
@@ -485,6 +514,43 @@ func (sp *ServerPeer) subscribeRecvMsg(subscription spMsgSubscription) {
 	sp.recvSubscribers[subscription] = struct{}{}
 }
 
+// A compile-time check to ensure that ServerPeer implements the query.Peer
+// interface.
+var _ query.Peer = (*ServerPeer)(nil)
+
+// SubscribeRecvMsg adds a OnRead subscription to the peer. All bitcoin
+// messages received from this peer will be sent on the returned channel. A
+// closure is also returned, that should be called to cancel the subscription.
+//
+// NOTE: Part of the query.Peer interface.
+func (sp *ServerPeer) SubscribeRecvMsg() (<-chan wire.Message, func()) {
```

Comment:
Why isn't the old `subscribeRecvMsg` sufficient?

Reply:
See commit message